src/bereikbaarheid/migrations/0007_alter_lastbeperking_lastbeperking_in_kg.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 4.1.10 on 2023-10-12 09:34
```

Comment:
in deze commit zit een nieuwe migratie zonder een wijziging in een model.py. Dat kan niet